tensorboard/plugins/hparams/backend_context_test.py

```diff
@@ -1185,6 +1185,175 @@ def test_experiment_from_tags_sorts_differed_hparams_first(self):
         )
         self.assertProtoEquals(expected_exp, actual_exp)
 
+    def test_experiment_from_runs_with_hparams_limit_no_differed_hparams(self):
+        self.session_1_start_info_ = """
+            hparams: [
+              {key: 'batch_size' value: {number_value: 100}},
+              {key: 'model_type' value: {string_value: 'LATTICE'}}
+            ]
+        """
+        self.session_2_start_info_ = """
+            hparams: [
+              {key: 'batch_size' value: {number_value: 100}},
+              {key: 'model_type' value: {string_value: 'LATTICE'}}
+            ]
+        """
+        self.session_3_start_info_ = """
+            hparams: [
+              {key: 'batch_size' value: {number_value: 100}},
+              {key: 'model_type' value: {string_value: 'LATTICE'}}
+            ]
+        """
+        expected_exp = """
+            hparam_infos: {
+              name: 'batch_size'
+              type: DATA_TYPE_FLOAT64
+              domain_interval: {
+                min_value: 100
+                max_value: 100
+              }
+              differs: false
+            }
+            hparam_infos: {
+              name: 'model_type'
+              type: DATA_TYPE_STRING
+              domain_discrete: {
+                values: [{string_value: 'LATTICE'}]
+              }
+              differs: false
+            }
+            metric_infos: {
+              name: {group: '', tag: 'accuracy'}
+            }
+            metric_infos: {
+              name: {group: '', tag: 'loss'}
+            }
+            metric_infos: {
+              name: {group: 'eval', tag: 'loss'}
+            }
+            metric_infos: {
+              name: {group: 'train', tag: 'loss'}
+            }
+        """
+        actual_exp = self._experiment_from_metadata()
```

Comment:
Maybe set include_metrics=False like the other tests?

Reply:
Done.